app-admin/kubelet/files/kubelet

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Wrapper for launching kubelet via rkt-fly stage1
+
+set -e
+
+if [ -z "${KUBELET_VERSION}" ]; then
+    echo "ERROR: must set KUBELET_VERSION"
+    exit 1
+fi
+
+KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+
+mkdir --parents /etc/kubernetes
+mkdir --parents /var/lib/docker
+mkdir --parents /var/lib/kubelet
+mkdir --parents /run/kubelet
+
+exec /usr/bin/rkt run \
+  --volume etc-kubernetes,kind=host,source=/etc/kubernetes \
+  --volume etc-ssl-certs,kind=host,source=/usr/share/ca-certificates \
+  --volume var-lib-docker,kind=host,source=/var/lib/docker \
+  --volume var-lib-kubelet,kind=host,source=/var/lib/kubelet \
+  --volume run,kind=host,source=/run \
```

Comment:
`kubelet` default docker to `/var/run`, if container don't have symlink from `/var/run` -> `/run` (like `docker://gcr.io/google_containers/hyperkube:v1.1.7`) it will fail